package.json

```diff
@@ -1,40 +1,40 @@
 {
   "name": "@babbel/miza-kinesis",
-  "version": "2.0.1",
+  "version": "2.1.0",
   "description": "Library to emit Events to Kinesis Events Queue",
   "main": "index.js",
   "types": "index.d.ts",
   "engines": {
-    "node": ">=12.14",
-    "npm": ">=6.13"
+    "node": ">=14",
```

Comment:
Does that mean that node 12 isn't supported anymore? 
If so, should miza-kinesis be updated to a major version?